aeppl/transforms.py

```diff
@@ -89,6 +91,13 @@ def transformed_logprob(op, value, *inputs, name=None, **kwargs):
     return logprob + jacobian
 
 
+class DefaultTransformSentinel:
+    pass
+
+
+DEFAULT_TRANSFORM = DefaultTransformSentinel()
+
+
 @local_optimizer(tracks=None)
 def default_transform(fgraph: FunctionGraph, node: Node) -> Optional[List[Node]]:
```

Comment:
Calling this opt `default_transform` feels wrong.

Reply:
Also don't you prefer the more specific return type hint?

Reply:
That is `Optional[List[TransformedRV]]`

Reply:
These are unrelated changes, but we should create some PRs for them.

Reply:
Done: https://github.com/aesara-devs/aeppl/issues/43